addons/mail/static/src/core/common/composer.js

```diff
@@ -120,6 +123,10 @@ export class Composer extends Component {
         this.suggestion = this.store.self.type === "partner" ? useSuggestion() : undefined;
         this.markEventHandled = markEventHandled;
         this.onDropFile = this.onDropFile.bind(this);
+        this.saveContentDebounced = useDebounced(this.saveInputToLocalStorage, 5000);
+        if (this.thread) {
+            useExternalListener(window, "beforeunload", this.saveInputToLocalStorage.bind(this));
+        }
```

Comment:
The `if ()` can be removed with previous comments on storing in local storage for all composers (including editing message). The key in local storage is simply `composer.localId` so no problem